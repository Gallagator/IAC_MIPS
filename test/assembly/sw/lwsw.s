    .text
	.align	2
	.globl	main
	.ent	main
main:
	.set	noreorder
    lui   $3, 0x7FFF
    addiu $3, $3, 1
    sw    $3, 0($0)
    jr $0
    lw    $2, 0($0)
.end	main
