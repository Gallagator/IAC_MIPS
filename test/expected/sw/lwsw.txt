0x7FFF0001